src/elements/Button/ButtonGroup.js

```diff
@@ -0,0 +1,79 @@
+import cx from 'classnames'
+import React, { PropTypes } from 'react'
+
+import META from '../../utils/Meta'
+import * as sui from '../../utils/semanticUtils'
+import {
+  getUnhandledProps,
+  useKeyOnly,
+} from '../../utils/propUtils'
+
+/**
+ * Button.Group
+ */
+function ButtonGroup(props) {
+  const {
+    basic, className, children, color, icon, size, vertical, widths,
+  } = props
+
+  const classes = cx('sd-button-group', 'ui',
+    size,
+    color,
+    useKeyOnly(basic, 'basic'),
+    useKeyOnly(icon, 'icon'),
+    useKeyOnly(vertical, 'vertical'),
+    className,
+    widths,
+    'buttons'
+  )
+
+  const rest = getUnhandledProps(ButtonGroup, props)
+
+  return (
+    <div className={classes} {...rest}>
+      {children}
+    </div>
+  )
+}
+
+
+ButtonGroup._meta = {
+  library: META.library.semanticUI,
+  name: 'ButtonGroup',
+  parent: 'Button',
+  type: META.type.element,
+  props: {
+    color: sui.colors,
+    size: sui.sizes,
+    widths: sui.widths,
+  },
+}
+
+
+ButtonGroup.propTypes = {
+  /** A button group can be less pronounced */
+  basic: PropTypes.bool,
+
+  /** Class names for custom styling. */
+  className: PropTypes.string,
+
+  /** */
+  children: PropTypes.any,
+
+  /** Groups can have a shared color */
+  color: PropTypes.oneOf(ButtonGroup._meta.props.color),
+
+  /** Groups can be formatted as icons */
+  icon: PropTypes.bool,
+
+  /** A button can have different sizes */
+  size: PropTypes.oneOf(ButtonGroup._meta.props.size),
+
+  /** Groups can be formatted to appear vertically */
+  vertical: PropTypes.bool,
+
+  /** Groups can have their widths divided evenly */
+  widths: PropTypes.oneOf(ButtonGroup._meta.props.widths),
```

Comment:
It is worth mentioning here that we also want to do away with the word number classes in this library.  Or at least offer a side by side alternative.  You can see this in the Grid Column where we use `width={3}` instead of `width='three'`.

We could do this easily by extending the `sui.widths` to include the numbers.  Then, extend the `numberToWord` util to include a map of words to words.  Finally, always use `utils/numberToWord.js` on the `width` prop values.

This way, users can use either numbers or words to define widths wherever they appear.